src/main/kotlin/bandcampcollectiondownloader/BandcampAPIConnector.kt

```diff
@@ -99,12 +100,12 @@ class BandcampAPIConnector constructor(private val bandcampUser: String, private
                                         .ignoreContentType(true)
                                         .timeout(timeout)
                                         .cookies(cookies)
+                                        .method(Method.POST)
                                         .requestBody("{\"fan_id\": $fanId, \"older_than_token\": \"$lastToken\"}")
-                                        .post()
+                                        .execute()
                             }, retries)
 
-
-                    val parsedCollectionData = gson.fromJson(theRest!!.wholeText(), ParsedCollectionItems::class.java)
+                    val parsedCollectionData = gson.fromJson(theRest!!.body(), ParsedCollectionItems::class.java)
```

Comment:
I wonder, isn't it only this change here that fixed your problem?

Reply:
`post()` returns a parsed `Document` and  `execute()` returns a `Response`. The `body()` methods are different for each. The former returns an `Element` and the latter returns a plain string.